### ISSUE TITLE
Strange behaviour

### ISSUE THREAD

**Issue by Diego02upa** [valinet/ExplorerPatcher#1836]
os:

Windows 11 Pro Edition
Version 22H2
OS build 22621.2134
Experience Windows Feature Experience Pack 1000.22659.1000.0
Explorer Patcher:

Version: 22621.1555.55.2

If I click on "This PC" or on a folder in the desktop, once out of 2 the taskbar crashes and restarts.

I don't know???

I then tried the latest version of Explorer Pacher (22621.1992.56.1); but it doesn't resolve.

This weird behavior I understand has something to do with the File manager.

Otherwise I don't have the problems that other users have or in any case with my use of the computer I haven't encountered problems like those of other users: I use the normal version of Windows 11 and I don't participate in the Windows insider program.

Thank you

**Comment by pyrates999** [comment]
Is it the same as this? https://github.com/valinet/ExplorerPatcher/issues/1805

**Comment by Diego02upa** [comment]
No, it's not the same thing.

I tried to change the style to "Window Switcher" and with all styles the anomalous behavior occurs both with Explorer Patcher version 22621.1555.55.2 and with version 22621.1992.56.1.


What is indicated in issue 1805 is the style in the "Taskbar" section.
Basically I had the "Windows 10" style and therefore with the problem described.

Then I tried the "Windows 11" style and the abnormal problem no longer occurs.

There is the problem of clicking with the right button on the Taskbar but it is solved by selecting the "Do not change the taskbar contect menu" item in the "Advanced" section.

Here, with the "Windows 11" Taskbar style you don't have the weather and I would like to use the "Windows 10" style (which uses weather).

Thank you
